strings/tcg/set_one.json

```diff
@@ -1089,46 +1089,46 @@
 			"id": "flamboyantborg",
 			"name": "Cyborg (Service Shell- Flamboyant)",
 			"desc": "Sometimes a cyborg just needs to show a bit of flamboyance, you know?",
-			"rules": "{$Asimov}, gains +2/+2 when it's the only card on your side of the field.",
+			"rules": "{$Asimov}, gains +2/+2 and loses {$Asimov} when it's the only card on your side of the field.",
 			"icon_state": "borg_serv_pink",
 			"power": "0",
 			"resolve": "1",
 			"faction": "Service",
 			"summoncost": "1",
 			"cardtype": "Creature",
-			"cardsubtype": "Silicon",
+			"cardsubtype": "Silicon Bar",
 			"rarity": "common",
 			"summon_icon_file": "icons/mob/silicon/robots.dmi",
 			"summon_icon_state": "brobot"
 		},
 		{
 			"id": "skirtborg",
-			"name": "Cyborg (Service Shell- Skirted)",
+			"name": "Cyborg (Service Shell - Skirted)",
 			"desc": "The Service Shell is intended to be the most human of the Cyborg Shells, due to its outwardly social role- none exemplify this better than the Skirted Shell, showing that even robots can't escape fashion norms.",
-			"rules": "{$Asimov}",
+			"rules": "{$Asimov} On Play: You heal 1 for every creature on your side of the field.",
```

Comment:
State units, does the borg heal 1 or does the player heal that much life?

Reply:
You the player. Clarified.